src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -3667,7 +3667,7 @@ public function set_attribute( $name, $value ): bool {
 			 *
 			 * @see https://html.spec.whatwg.org/#attributes-3
 			 */
-			$escaped_new_value = in_array( $comparable_name, wp_kses_uri_attributes() ) ? esc_url( $value ) : esc_attr( $value );
+			$escaped_new_value = in_array( $comparable_name, wp_kses_uri_attributes(), true ) ? esc_url( $value ) : esc_attr( $value );
```

Comment:
@xknown, @aaronjorbin, @audrasjb This was added in https://core.trac.wordpress.org/changeset/58473 where you're propped. Can you confirm whether this loose `in_array` was intentional?

It seems like we want strict as we expect to be comparing strings, but I want to confirm first.

If we'd prefer to keep the loose comparison that's fine, I'd like to apply a different change because PHPCS is flagging this line as a warning now.